terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 1: 29817 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 2: 29819 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 3: 29821 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 4: 29823 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 5: 29825 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 6: 29827 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 7: 29829 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 8: 29831 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 9: 29833 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 10: 29835 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 11: 29837 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 12: 29839 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 13: 29842 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 14: 29845 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 15: 29847 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 16: 29849 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 17: 29851 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 18: 29853 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 19: 29855 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 20: 29857 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 21: 29859 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 22: 29862 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 23: 29864 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 24: 29866 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 25: 29868 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 26: 29870 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 27: 29872 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 28: 29874 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 29: 29876 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 30: 29878 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 31: 29880 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 32: 29882 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 33: 29884 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 34: 29886 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 35: 29888 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 36: 29890 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 37: 29892 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 38: 29894 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 39: 29896 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 40: 29898 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 41: 29900 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 42: 29902 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 43: 29904 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 44: 29906 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 45: 29908 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 46: 29910 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 47: 29912 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 48: 29914 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 49: 29916 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 50: 29918 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 51: 29920 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 52: 29922 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 53: 29924 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 54: 29926 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 55: 29928 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 56: 29930 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 57: 29932 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 58: 29934 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 59: 29936 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 60: 29938 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 61: 29940 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 62: 29942 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 63: 29944 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 64: 29946 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 65: 29948 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 66: 29950 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 67: 29952 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 68: 29954 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 69: 29956 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 70: 29958 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 71: 29960 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 72: 29962 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 73: 29964 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 74: 29966 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 75: 29968 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 76: 29970 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 77: 29972 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 78: 29984 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 79: 29986 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 80: 29989 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 81: 29991 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 82: 29993 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 83: 29995 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 84: 29997 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 85: 29999 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 86: 30001 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 87: 30003 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 88: 30005 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 89: 30007 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 90: 30009 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 91: 30011 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 92: 30013 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 93: 30017 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 94: 30019 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 95: 30021 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 96: 30023 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 97: 30025 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 98: 30027 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 99: 30029 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 100: 30032 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 101: 30034 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/list/5.neglite.program'
./scripts/neg/ijcai19/list-val/list-NEGLITE-1-2-a-val.sh: line 102: 30036 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/list/domain.pddl domains/neg/list/list-1-neg-a.pddl $1
